build_bundle_image.sh

```diff
@@ -273,6 +282,31 @@ function push_docker_image {
 	fi
 }
 
+function print_help {
+	echo "Usage: ${0} --push"
+	echo ""
+	echo "The script reads the following environment variables:"
+	echo ""
+	echo "    LIFERAY_DOCKER_DEVELOPER_MODE (optional): If set to \"true\", all local images will be deleted before building a new one"
+	echo "    LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES (optional): Elasticsearch remote server network addresses"
+	echo "    LIFERAY_DOCKER_FIX_PACK_URL (optional): URL to a fix pack"
+	echo "    LIFERAY_DOCKER_HUB_TOKEN (optional): Docker Hub token to log in automatically"
+	echo "    LIFERAY_DOCKER_HUB_USERNAME (optional): Docker Hub username to log in automatically"
+	echo "    LIFERAY_DOCKER_IMAGE_PLATFORMS (optional): Comma separated Docker image platforms to build when the \"push\" parameter is set"
+	echo "    LIFERAY_DOCKER_LICENSE_API_HEADER (required for DXP): API header used to generate the trial license"
+	echo "    LIFERAY_DOCKER_LICENSE_API_URL (required for DXP): API URL to generate the trial license"
+	echo "    LIFERAY_DOCKER_RELEASE_FILE_URL (required): URL to a Liferay bundle"
+	echo "    LIFERAY_DOCKER_REPOSITORY (optional): Docker repository"
+	echo "    LIFERAY_DOCKER_SLIM (optional): If set to \"true\", the image will be the slim variant"
+	echo ""
+	echo "Example: LIFERAY_DOCKER_RELEASE_FILE_URL=files.liferay.com/private/ee/portal/7.2.10/liferay-dxp-tomcat-7.2.10-ga1-20190531140450482.7z ${0} push"
+	echo ""
+	echo "Example: LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES='[\"http://es-node1:9200\",\"http://es-node2:9201\"]' LIFERAY_DOCKER_RELEASE_FILE_URL=files.liferay.com/private/ee/portal/7.2.10/liferay-dxp-tomcat-7.2.10-ga1-20190531140450482.7z LIFERAY_DOCKER_SLIM=true ${0} push"
```

Comment:
@kiwm, can you allow the user type the network addresses without the additional characters: `[`, `'`, `"`, `]` ?

The user should only type `LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES=http://es-node1:9200, http://es-node2:9201`. Consequently, the additional characters can be added by function `prepare_slim_image`.